CMakeLists.txt

```diff
@@ -365,6 +365,72 @@ file(GLOB_RECURSE CRPROPA_INCLUDES RELATIVE ${CMAKE_SOURCE_DIR} include/*.h)
 include_directories(include ${CRPROPA_EXTRA_INCLUDES})
 
 add_library(crpropa SHARED
+	src/base64.cpp
+	src/Candidate.cpp
+	src/Clock.cpp
+	src/Common.cpp
+	src/Cosmology.cpp
+	src/EmissionMap.cpp
+	src/Geometry.cpp
+	src/GridTools.cpp
+	src/GridTurbulence.cpp
+	src/Module.cpp
+	src/ModuleList.cpp
+	src/ParticleID.cpp
+	src/ParticleMass.cpp
+	src/ParticleState.cpp
+	src/PhotonBackground.cpp
+	src/PhotonPropagation.cpp
+	src/ProgressBar.cpp
+	src/Random.cpp
+	src/Source.cpp
+	src/Variant.cpp
+	src/module/Boundary.cpp
+	src/module/BreakCondition.cpp
+	src/module/DiffusionSDE.cpp
+	src/module/EMCascade.cpp
+	src/module/EMDoublePairProduction.cpp
+	src/module/EMInverseComptonScattering.cpp
+	src/module/EMPairProduction.cpp
+	src/module/EMTripletPairProduction.cpp
+	src/module/ElasticScattering.cpp
+	src/module/ElectronPairProduction.cpp
+	src/module/HDF5Output.cpp
+	src/module/NuclearDecay.cpp
+	src/module/Observer.cpp
+	src/module/Output.cpp
+	src/module/OutputShell.cpp
+	src/module/ParticleCollector.cpp
+	src/module/PhotoDisintegration.cpp
+	src/module/PhotoPionProduction.cpp
+	src/module/PhotonEleCa.cpp
+	src/module/PhotonOutput1D.cpp
+	src/module/PropagationCK.cpp
+	src/module/PropagationBP.cpp
+  src/module/Redshift.cpp
+	src/module/RestrictToRegion.cpp
+	src/module/SimplePropagation.cpp
+	src/module/SynchrotronRadiation.cpp
+	src/module/TextOutput.cpp
+	src/module/AdiabaticCooling.cpp
+	src/module/Tools.cpp
+	src/magneticField/JF12Field.cpp
+	src/magneticField/JF12FieldSolenoidal.cpp
+	src/magneticField/MagneticField.cpp
+	src/magneticField/MagneticFieldGrid.cpp
+  src/magneticField/PolarizedSingleModeMagneticField.cpp
+	src/magneticField/PT11Field.cpp
+  src/magneticField/TF17Field.cpp
+	src/magneticField/ArchimedeanSpiralField.cpp
+	src/advectionField/AdvectionField.cpp
+	src/massDistribution/Nakanishi.cpp
+	src/massDistribution/Massdistribution.cpp
+	src/massDistribution/Ferriere.cpp
+	src/massDistribution/Cordes.cpp
+	src/massDistribution/ConstantDensity.cpp
+
+	${CRPROPA_EXTRA_SOURCES}
+=======
```

Comment:
Line 433 is problematic. @asavelie : you probably forgot to remove the "===" after solving the conflicts.